pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go

```diff
@@ -103,6 +103,13 @@ func (mutator *VMIsMutator) Mutate(ar *v1beta1.AdmissionReview) *v1beta1.Admissi
 			log.Log.V(2).Infof("Failed to set HyperV dependencies: %s", err)
 		}
 
+		// Do some specific setting for Arm64 Arch. It should put before SetObjectDefaults_VirtualMachineInstance
+		log.Log.V(4).Info("Apply Arm64 specific setting")
+		err = webhooks.SetVirtualMachineInstanceArm64Setting(newVMI)
+		if err != nil {
+			// We should leave rejection to be performed later on in the validating webhook, and continue here.
```

Comment:
maybe it is fine to abort early, cannot get the point why it will continue here ...

Reply:
if SetVirtualMachineInstanceArm64Setting fails, it's due to a validation error, which will get caught in the validation webhook after mutation finishes.